exec/memory/base/src/main/java/io/netty/buffer/PooledByteBufAllocatorL.java

```diff
@@ -59,7 +59,7 @@ public PooledByteBufAllocatorL(MetricRegistry registry) {
 
   public UnsafeDirectLittleEndian allocate(int size) {
     try {
-      return allocator.directBuffer(size, size);
+      return allocator.directBuffer(size, Integer.MAX_VALUE);
```

Comment:
The RPC uses ensureWritable to expand buffers. Buffers are expandable to maxCapacity. We were coding maxCapacity == capacity which meant that buffers weren't expandable and the RPC layer would error out. This fixes this issue.